src/main/java/uk/gov/hmcts/reform/professionalapi/service/impl/OrganisationServiceImpl.java

```diff
@@ -129,6 +136,18 @@ public Organisation saveOrganisation(Organisation organisation) {
         return persistedOrganisation;
     }
 
+    public void addDefaultMfaStatusToOrganisation(Organisation organisation) {
+
+        OrganisationMfaStatus organisationMfaStatus = new OrganisationMfaStatus();
+        organisationMfaStatus.setOrganisation(organisation);
+        organisationMfaStatus.setMfaStatus(EMAIL);
```

Comment:
I think default value is already been set in the db. is it required to set again?

Reply:
Yes, you're right this is not required. I've just made a change and defaulted email in the entity class instead. Please kindly review the change @SujeethKumarAri